README.md

```diff
@@ -51,9 +58,10 @@ autoUpdater.setFeedURL(feed)
 As the final step, check for updates. The example below will check every minute:
```

Comment:
> The example below will check every minute:

Every ten minutes, so you can also get rid of the comment.

Reply:
Thanks, @malept 👀 

Hotfixed to master with https://github.com/electron/update.electronjs.org/commit/20d301f278a219f752b8d474c79b8ab0f059e428